direct/algorithms/__init__.py

```diff
@@ -0,0 +1,2 @@
+# coding=utf-8
+# Copyright (c) DIRECT Contributors
```

Comment:
Pylint likely requires you to give a module description. Can you try to run `pylint` on this submodule?